app/helpers/admin_helper.rb

```diff
@@ -0,0 +1,6 @@
+module AdminHelper
+  def admin_page?
+    url = request.original_url.split("/")
+    url[3] == "admin" || url[4] == "admin"
```

Comment:
```suggestion
    url(3,2).include?("admin")
```
Em thử như này xem được không em

Reply:
dạ không được ạ